health_check/backends/base.py

```diff
@@ -44,6 +46,23 @@ def status(self):
 
         return self._status
 
+    def get_logger(self, logger):
+        if logger:
+            return logger
+        if hasattr(self, "logger"):
+            return self.logger
+        return getLogger(self.__name__)
```

Comment:
We can also use self.__module__ here which should return the child class module from where the logging is done from. This is somewhat equivalent to __name__ used currently in the different subclassed health checking backend modules.